src/app/utils/getComments.js

```diff
@@ -0,0 +1,7 @@
+export default async function getComments(id) {
+  const itemUrl = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/FBTV7LVQNoUbR7Qxl6jA/comments?item_id=${id}`;
+  const response = await fetch(itemUrl);
+  const data = await response.json();
+  return data;
+  // this returns comment, username and creation_date
+}
```

Comment:
- [ ] - [ ] Kindly use `Es6` syntax for creating a function because it was specified in the project that you should use `Es6`. It would look something like this:

`export default const functionName = (parameters) => {}`